### PR TITLE
SEC-51 filing-history-data-api CORS

### PR DESCRIPTION
**Pull request by NeilShadrach**
## Describe the changes
filing-history-data-api CORS
### Related Jira tickets
SEC-51 
[<Jira Ticket ID>](<Jira URL>)

### General

- [ ] Is the PR as small as it can be?
- [ ] Has the Jira ticket been updated with any relevant comments?
- [ ] Is the `README` up to date?
- [ ] Has the `POM` been updated for the latest versions of dependencies?
- [ ] Has the code been double-checked against `main`?
- [ ] Does the code adhere standards in this repository, including SonarQube checks?

### Testing

- [ ] Do the code changes have unit and integration tests with code coverage > 80%?
- [ ] Has the code been tested locally and/or passed the API Karate tests on Tilt?
- [ ] Have mandatory manual test cases been run?
- [ ] Are extra Karate tests required?
- [ ] Are all the test data resources up to date?

### Release preparation

_Where possible, add links to the respective Jira tickets when an item is checked_

- [ ] Have these changes been included in a release ticket?
- [ ] Are changes required to the `docker-chs-development` deployment or test data?
- [ ] Are any changes required to the environment added to deployment repositories?

## Notes to the tester

_Add any testing hints or instructions here._

### CODE REVIEW

**Review by hselvakumar — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <name>filing-history-data-api</name>
 
   <artifactId>filing-history-data-api</artifactId>
-  <version>latest</version>
+  <version>unversioned</version>
```
Comment (hselvakumar) [inline pom.xml:15]: is this by mistake ?

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: pom.xml (hunk 1 above)
Comment (NeilShadrach) [reply pom.xml:15]: That was last week. I no longer remember. I'll take it out and retest

Comment (NeilShadrach) [reply pom.xml]: Dropped the L15 change

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Pull request merged**